### PR TITLE
More explicit sqls instructions +mssql dsn example

### PR DESCRIPTION
**Pull request by unhammer**

### CODE REVIEW

File: docs/manual-language-docs/lsp-sqls.md [hunk 1]
```diff
@@ -2,10 +2,28 @@
 
 
 ``` emacs-lisp
+(add-hook 'sql-mode 'lsp-deferred)
```
Comment (yyoncho) [inline docs/manual-language-docs/lsp-sqls.md:5]: nit: `lsp` is the default.

Comment (unhammer) [reply docs/manual-language-docs/lsp-sqls.md:5]: wops!

File: docs/manual-language-docs/lsp-sqls.md [hunk 2]
```diff
@@ -2,10 +2,28 @@
 
 
 ``` emacs-lisp
+(add-hook 'sql-mode 'lsp)
```
Comment (yyoncho) [inline docs/manual-language-docs/lsp-sqls.md:5]: ... and it should be `sql-mode-hook`

**Review by yyoncho — APPROVED** [review]

File: docs/manual-language-docs/lsp-sqls.md (hunk 2 above)
Comment (unhammer) [reply docs/manual-language-docs/lsp-sqls.md]: doh! I switched to use-package `:hook ((sql-mode . lsp-deferred))` for myself but didn't want to complicate things … fixed.

**Comment by yyoncho** [comment]
Thank you

**Pull request merged**